ci-operator/step-registry/ipi/conf/gcp/user-labels-tags/ipi-conf-gcp-user-labels-tags-commands.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+CONFIG="${SHARED_DIR}/install-config.yaml"
+PATCH="${SHARED_DIR}/user-labels-tags.yaml.patch"
+
+# user labels
+i=0
+printf '%s' "${USER_LABELS:-}" | while read -r KEY VALUE || [ -n "${KEY}" ]
+do
+  yq-go write -i "${PATCH}" "platform.gcp.userLabels[$i].key" "${KEY}"
+  yq-go write -i "${PATCH}" "platform.gcp.userLabels[$i].value" "${VALUE}"
+  i=$(( $i + 1))
+done
```

Comment:
From azure, aws codes, it is using something like:
```
yq-go write -i "${CONFIG}" "platform.aws.userTags.${TAG}" "${VALUE}"
```
Does it work on gcp, too, right?

Reply:
I'd like to create a PATCH file and finally merge with CONFIG, because in case of GCP, userLabels and userTags are list of objects, where an index is needed when providing key/value/parentID. That's it, thanks!

Reply:
Got it.